src/picongpu/include/initialization/InitialiserController.hpp

```diff
@@ -94,25 +94,18 @@ class InitialiserController : public IInitPlugin
     {
         if (Environment<simDim>::get().GridController().getGlobalRank() == 0)
         {
-            log<picLog::PHYSICS >("max weighting %1%") % NUM_EL_PER_PARTICLE;
-
             log<picLog::PHYSICS >("Courant c*dt <= %1% ? %2%") %
                                  (1./math::sqrt(INV_CELL2_SUM)) %
                                  (SPEED_OF_LIGHT * DELTA_T);
 
-            if (gasProfile::GAS_ENABLED)
```

Comment:
I got an idea because we talked about his line: we could add an `omega_p"species_short_name" * dt` for each species here - that would be awesome